ucsb-cs156-f24/team02-f24-07
# LD - Create Page

Closes #22 

To Test: 
Click Submit, which should give you errors.
Then, enter in strings, and then click enter, then navigate to swagger, and the inputted organization should show up.

https://67227cb17ca58541ac0366fc-yynlmohayw.chromatic.com/?path=/docs/pages-ucsborganization-ucsborganizationcreatepage--docs
